packages/hash/frontend/package.json

```diff
@@ -94,7 +81,6 @@
     "react-transition-group": "4.4.2",
     "rooks": "5.7.1",
     "setimmediate": "1.0.5",
-    "slugify": "1.6.5",
```

Comment:
Used in shared for https://github.com/hashintel/hash/blob/d067f2fec5013cfac4b385a274977bb532263849/packages/hash/shared/src/types.ts#L9-L10i believe

Reply:
We still have slugify in shared